schemas/test/en/test_progress_value_source_section_enabled_hub_complex.json

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
What is complex about it?. Can you add some description on what should happen etc.

Reply:
Added description